tests/kernel/wmp_simple.pm

```diff
@@ -29,6 +29,10 @@ sub run {
     my $stressng_mem = get_required_var('WMP_STRESS_MEM');
     my $stressng_repo = get_var('WMP_STRESS_REPO', "https://download.opensuse.org/repositories/benchmark/SLE_15_SP3/benchmark.repo");
 
+    my $sid = get_required_var('INSTANCE_SID');
+    my $instance_id = get_required_var('INSTANCE_ID');
```

Comment:
I suppose by the use of `get_required_var()` in these 2 lines, that this module only applies to SAP workload tests. Correct?

Reply:
Yes, that's the only intended use for this module. If we need to use it for different purpose we still can change this later.